contentcuration/contentcuration/db/models/aggregates.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models.aggregates import Aggregate
+from django.db.models.fields import BooleanField
+
+
+class BoolOr(Aggregate):
```

Comment:
Any reason to introduce this rather than just use https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/aggregates/#boolor ?

Reply:
Ah totally missed it. I only checked the common aggregate file. I'll have to see if there's anything else I duplicated.